jpa/subsystem/src/main/java/org/jboss/as/jpa/subsystem/JPADeploymentDefinition.java

```diff
@@ -0,0 +1,40 @@
+package org.jboss.as.jpa.subsystem;
+
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.SimpleResourceDefinition;
+import org.jboss.as.controller.descriptions.ModelDescriptionConstants;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
+import org.jboss.dmr.ModelNode;
+
+import static org.jboss.as.jpa.subsystem.JPADefinition.DEFAULT_DATASOURCE;
+import static org.jboss.as.jpa.subsystem.JPADefinition.DEFAULT_EXTENDEDPERSISTENCE_INHERITANCE;
+
+public class JPADeploymentDefinition extends SimpleResourceDefinition {
+
+    JPADeploymentDefinition() {
+        super(getParameters());
+    }
+
+    private static Parameters getParameters() {
+        Parameters result = new Parameters(PathElement.pathElement(ModelDescriptionConstants.SUBSYSTEM, JPAExtension.SUBSYSTEM_NAME),
```

Comment:
I would recommend creating a static PathElement constant within this class that references the subsystem path, rather than calculating it multiple times.